spiders.html

```diff
@@ -95,7 +95,14 @@
         return "rgb(" + red + "," + green + ",0)";
     }
 
+    const LINK_FORMAT_OPTIONS = [
+        ['geojson', 'Feature GeoJSON'],
+        ['statistics', 'Spider stats JSON'],
+        ['logs', 'Spider error logs'],
+    ];
```

Comment:
```suggestion
    const LINK_FORMAT_OPTIONS = {
        geojson: 'Feature GeoJSON',
        statistics: 'Spider stats JSON',
        logs: 'Spider error logs',
    };
```

(etc.)

Reply:
I don't think it needs to be a number anymore, for what it's worth, but you can use the `selectedIndex` property of the select element.

Inefficient is not relevant for a tiny object that's used to generate some HTML, this is a readability concern. It's also an opinion, either way is really fine. Expressing it as an object makes it more obvious that the things on the left hand side are keys.